overrides/build-systems.json

```diff
@@ -7228,7 +7228,18 @@
     "setuptools"
   ],
   "httpcore": [
-    "setuptools"
+    {
+      "buildSystem": "hatch-fancy-pypi-readme",
+      "from": "0.18.0"
+    },
+    {
+      "buildSystem": "hatchling",
+      "from": "0.18.0"
+    },
+    {
+      "buildSystem": "setuptools",
+      "until": "0.18.0"
```

Comment:
`until` entries should probably be before `from` entries, to keep the sequence is ascending order of versions. Other packages seem to have been done that way.

Reply:
Thanks for your input! I actually copied the entry for `httpx` a couple of lines below, and it's ordered the same way (`from` entries before `until` entries), so maybe we could try to ensure consistency across the entire file?

Reply:
Consistency would be great. There is already a sorting step in CI; I've filed an [issue to strengthen it](https://github.com/nix-community/poetry2nix/issues/1320).